src/inverseRobot.js

```diff
@@ -24,7 +24,18 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const keys = Object.keys(robot);
+  const newRobot = {};
+
+  for (const key of keys) {
+    if (newRobot.hasOwnProperty(robot[key])) {
```

Comment:
```suggestion
  const newRobot = {};

  for (const key in robot) {
    if (newRobot.hasOwnProperty(robot[key])) {
```

using loop 'for...in' is much shorter and more simple here